test/memoryPool.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef UMF_TEST_MEMORY_POOL_OPS_HPP
 #define UMF_TEST_MEMORY_POOL_OPS_HPP
 
+#include "malloc_compliance_tests.hpp"
```

Comment:
Could you move all includes to #ifndef guards?

Reply:
Done